examples/azure/terraform-azure-cosmosdb-example/main.tf

```diff
@@ -0,0 +1,88 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY AN AZURE VIRTUAL MACHINE
+# This is an example of how to deploy an Azure Virtual Machine with the minimum network resources.
+# ---------------------------------------------------------------------------------------------------------------------
+# See test/azure/terraform_azure_example_test.go for how to write automated tests for this code.
+# ---------------------------------------------------------------------------------------------------------------------
+
+provider "azurerm" {
+  version = "~> 2.29"
+  features {}
+}
+
+
+# ---------------------------------------------------------------------------------------------------------------------
+# PIN TERRAFORM VERSION TO >= 0.12
+# The examples have been upgraded to 0.12 syntax
+# ---------------------------------------------------------------------------------------------------------------------
+
+terraform {
+  # This module is now only being tested with Terraform 0.13.x. However, to make upgrading easier, we are setting
+  # 0.12.26 as the minimum version, as that version added support for required_providers with source URLs, making it
+  # forwards compatible with 0.13.x code.
+  required_version = ">= 0.12.26"
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A RESOURCE GROUP
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "azurerm_resource_group" "rg" {
+  name     = var.resource_group_name
+  location = var.location
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A COSMOSDB ACCOUNT
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "azurerm_cosmosdb_account" "test" {
+  name                = var.cosmosdb_account_name
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  offer_type          = "Standard"
+  kind                = "GlobalDocumentDB"
+
+  consistency_policy {
+    consistency_level       = "Session"
+    max_interval_in_seconds = 5
+    max_staleness_prefix    = 100
+  }
+
+  geo_location {
+    location          = azurerm_resource_group.rg.location
+    failover_priority = 0
+  }
+}
+
+resource "azurerm_cosmosdb_sql_database" "testdb" {
+  name                = "testdb"
+  throughput          = 400
+  resource_group_name = azurerm_resource_group.rg.name
+  account_name        = azurerm_cosmosdb_account.test.name
+}
+
+resource "azurerm_cosmosdb_sql_container" "container1" {
+  name                = "test-container-1"
+  throughput          = 400
```

Comment:
consider making throughput a variable in variables.tf so that in your test you can assert the expected value you passed in.

Reply:
Makes sense, done.